### PR TITLE
added unit test

### PR DESCRIPTION
**Pull request by xRuBYN**

### CODE REVIEW

File: src/main/test/java/com/endava/internship/collections/BTreeTest.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    @Test
+    public void whenWeDeleteAnExistentElementInOurSetMethodShouldReturnTrue() {
+        tree.addNode(student1);
+        tree.addNode(student3);
+        tree.addNode(student4);
+        tree.addNode(student5);
+        assertTrue(tree.delete(student4));
+        assertTrue(tree.delete(student3));
+        assertTrue(tree.delete(student1));
+    }
```
Comment (meedvv379) [inline src/main/test/java/com/endava/internship/collections/BTreeTest.java]: could you please check and size in all tests where you have delete method?

File: src/main/test/java/com/endava/internship/collections/BTreeTest.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
+package com.endava.internship.collections;
+
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import java.time.LocalDate;
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (meedvv379) [inline src/main/test/java/com/endava/internship/collections/BTreeTest.java]: please don't use wildcard import

File: src/main/test/java/com/endava/internship/collections/BTreeTest.java [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+public class BTreeTest {
+
+    private Student student1;
+    private Student student2;
```
Comment (meedvv379) [inline src/main/test/java/com/endava/internship/collections/BTreeTest.java]: student2 is not used, if you don't want to use delete it please

Comment (xRuBYN) [reply src/main/test/java/com/endava/internship/collections/BTreeTest.java]: fixed

File: src/main/test/java/com/endava/internship/collections/BTreeTest.java (hunk 1 above)
Comment (xRuBYN) [reply src/main/test/java/com/endava/internship/collections/BTreeTest.java]: fixed

File: src/main/test/java/com/endava/internship/collections/BTreeTest.java (hunk 2 above)
Comment (xRuBYN) [reply src/main/test/java/com/endava/internship/collections/BTreeTest.java]: resolve

**Review by meedvv379 — APPROVED** [review]

**Pull request merged**